WPILib.Extras/SubsystemCommand.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using WPILib.Commands;
+
+namespace WPILib.Extras
+{
+    /// <summary>
+    /// A <see cref="ActionCommand"/> will run an <see cref="Action"/> when it is
```

Comment:
Also I'm very confused on what the actual use case for this is. If this is just a hacky way to add a requirement to a command I don't like it. I would need an explanation on how this would be used. Better XML docs might help with this.